### ISSUE TITLE
Add air pressure?

### ISSUE THREAD

**Issue by Giorgiofox** [lucacri/homebridge-http-temperature-humidity#26]
Hello, this is not an issue but a feature request:

is possible to add air pressure and why not wind speed and direction?

**Comment by lucacri** [comment]
I don't have any air/wind device, do you? I'll be open to merge any PR

**Comment by Giorgiofox** [comment]
Yes, I have these sensors. I'm using a (Davis Vantage Pro 2) weather station, with software WVIEW, and I can generate JSON file. 
Actually I think these are the most useful parameters (with measure units)

Outside Temperature:	-1.6 C
Inside Temperature:	19.6 C
Wind Chill:	-1.6 C
Outside Humidity:	92 %
Inside Humidity:	58 %
Barometer:	1038.8  mb +
Wind:	NNW  at   0.0  km/h
High Wind:	4.8 km/h at 06:55
Today's Rain:	0.00  mm
Rain Rate:	0.00  mm/h
Raining: No

I think that integrate these is really interesting for everyone have a weather station.
Also the Wind sensor can help to automate some script, for example "do something when wind reach this value".

Maybe we can start adding Barometer (air pressure). I will write the script to generate JSON, and publish here.

**Comment by paulkoene** [comment]
For me this would also be a great added feature. I already have scripts setup to return the following format. It would also be great if I could launch a local script instead of having to go through a webserver process. Finally it would be cool if I could also attach my homes energy usage (in Watt) to the script. Does anybody by any chance know why the sensor readings can't be used for automation in homekit? It would be cool if I could use homekit to control the central heating.

{ "temperature": 19.6, "humidity": 62.5, "pressure": 103669 }

**Comment by DanielWeeber** [comment]
Is there a Service in Hap-NodeJS for Airpressure? Just looking for it, cannot find it.
